### PR TITLE
Internal AppTP support

### PR DESCRIPTION
**Pull request by samsymons**
Task/Issue URL: https://app.asana.com/0/414235014887631/1204167883757917/f
Tech Design URL:
CC: @SlayterDev 

**Description**:

See Asana task for context

**Steps to test this PR**:
1. See Asana task for testing steps, specifically [this subtask](https://app.asana.com/0/0/1204342309532634/f)

**Copy Testing**:

* [x] Use of correct apostrophes in new copy, ie `’` rather than `'`

**Orientation Testing**:

* [x] Portrait
* [x] Landscape

**Device Testing**:

* [ ] iPhone SE (1st Gen)
* [ ] iPhone 8
* [ ] iPhone X
* [x] iPhone 14 Pro
* [ ] iPad

**OS Testing**:

* [ ] iOS 14
* [ ] iOS 15
* [x] iOS 16

**Theme Testing**:

* [x] Light theme
* [x] Dark theme

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

**Review by afterxleep — COMMENTED** [review]
Hey @samsymons.   👋

I checked the last few commits with the integration changes, and it looks good overall, and can also confirm that AppTP is available in the app only after login.

Any particulars you want me to look into?

**Comment by afterxleep** [comment]
Also, I'm unable to test AppTp on device as app is not building in 14.3 due to the Kingfisher issue, and all my devices have been updated to 16.4.x.  I understand you were working on the KF updates?  Got a branch I could integrate temporarily to test functionality on device?

**Comment by samsymons** [comment]
@afterxleep That's right, the Kingfisher change is still open here: https://github.com/duckduckgo/iOS/pull/1614

I'll be trying to get that merged in the next day or so, after which I'll pull `develop` back into this branch and you should be good to go. I'll ping you once that's ready. Thanks!

**Comment by samsymons** [comment]
@afterxleep This should be good to go now. Let me know if you run into trouble building for your device in Xcode 14.3!

**Review by afterxleep — APPROVED** [review]
Looking pretty good 💪 

Given the PR size, I understand these changes were previously reviewed on a feature branch, so I focused on the latest commits that look related to the task description.

**Here's what I tested:**

✅  Enabled tracking protection and left it running for at least two hours on my device, confirming trackers are being blocked (as per the list says)
✅  Manually disabled the 'VPN' to confirm it reconnects automatically
✅  Disabled 'Connect on demand' and manually disconnected the 'VPN' profile to confirm AppTp shows as disabled but reconnects when enabling the feature from the app 
✅ Went to the VPN settings and deleted the 'VPN' profile to confirm AppTP is disabled in the app and the profile gets created adequately after enabling it again.
✅ 'Show additional tracking info' works as expected. (As the list of trackers grows, reaching that setting might require plenty of scroll)
✅ "Something not working?" The option shows the form, and I was able to submit (Did not get any confirmation, but I guess that's part of a later iteration)
✅ Sorting functionality (It works, but I find it a bit confusing that the 'Today' header still appears when sorting by count.  Should it be only there when sorting by date?)
✅ Different screens orientations and sizes on iPhone and iPad


ℹ️  **Note:** I have not tested on iOS 14 and 15 as I don't have devices running those versions ATM


For reference, this is what I see:

Screenshot 2023-04-12 at 10 17 42
Screenshot 2023-04-12 at 10 18 16
IMG_3829

File: DuckDuckGo/AppDelegate.swift [hunk 1]
```diff
@@ -152,6 +153,15 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             legacyStorage?.removeStore()
             WidgetCenter.shared.reloadAllTimelines()
         }
+
+        appTrackingProtectionDatabase.loadStore { context, error in
+            guard context != nil else {
+                // TODO: Fire pixel here, then sleep for 1 second to allow it to send
```
Comment (samsymons) [inline DuckDuckGo/AppDelegate.swift]: I'll update this to fire a pixel

File: .swiftlint.yml [hunk 2]
```diff
@@ -80,3 +80,5 @@ excluded:
   - fastlane/SnapshotHelper.swift
   - vendor
   - LocalPackages/*/Package.swift
+  - LocalPackages/DuckUI/Package.swift
```
Comment (samsymons) [inline .swiftlint.yml]: We can remove this in favour of the line above it

**Pull request merged**